### PR TITLE
(V2.2 1/3) Add grade attributes to accredible DB table

### PR DESCRIPTION
**Pull request by octaviorequena**
Source: [NTGR-291](https://accredible.atlassian.net/browse/NTGR-291)

### CODE REVIEW

File: db/install.xml [hunk 1]
```diff
@@ -17,10 +17,13 @@
         <FIELD NAME="timecreated" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
         <FIELD NAME="certificatename" TYPE="char" LENGTH="255" NOTNULL="true" SEQUENCE="false"/>
         <FIELD NAME="groupid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false"/>
+        <FIELD NAME="includegradeattribute" TYPE="int" LENGTH="1" NOTNULL="false" DEFAULT="0" SEQUENCE="false"/>
```
Comment (kei178) [inline db/install.xml:20]: I'd add this field after `completionactivities` for consistency with `upgrade.php`.

File: db/upgrade.php [hunk 2]
```diff
@@ -130,5 +130,39 @@ function xmldb_accredible_upgrade($oldversion=0) {
...
+        $field = new xmldb_field('gradeattributekeyname', XMLDB_TYPE_CHAR, '255', null, null, null,
+            null, 'gradeattributegradeitemid');
+
+        // Conditionally launch add field groupid.
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
+        }
+
+        // Accredible savepoint reached.
+        upgrade_mod_savepoint(true, 2022053000, 'accredible');
```
Comment (kei178) [inline db/upgrade.php:164]: Please update the release instructions in Jira to update this version to the release version.

**Review by kei178 — APPROVED** [review]
@octaviorequena - Looks good to me. Nice work 🚀  Please just update the release instructions in Jira.

File: db/upgrade.php [hunk 3]
```diff
@@ -130,5 +130,39 @@ function xmldb_accredible_upgrade($oldversion=0) {
 
     }
 
+    if ($oldversion < 2022053000) {
+        $table = new xmldb_table('accredible');
+
+        // Define field includegradeattribute to be added to accredible.
+        $field = new xmldb_field('includegradeattribute', XMLDB_TYPE_INTEGER, '1', null, null, null, '0',
```
Comment (kei178) [inline db/upgrade.php:137]: Should we add `XMLDB_NOTNULL` to `includegradeattribute` since it's a boolean with a default value?

**Review by naveenbalachandar — APPROVED** [review]

**Pull request merged**